src/Riok.Mapperly.Abstractions/Riok.Mapperly.Abstractions.csproj

```diff
@@ -7,4 +7,11 @@
   <ItemGroup>
     <Using Include="Riok.Mapperly.Abstractions" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>

```

Comment:
Do we need the `IncludeAssets` here? Shouldn't all the assets be private assets, so that they aren't included in the NuGet output?